CMakeLists.txt

```diff
@@ -27,36 +27,34 @@ cet_cmake_module_directories(Modules BINARY)
 
 find_package(artg4tk REQUIRED EXPORT)
 find_package(art REQUIRED EXPORT)
-find_package(art_root_io REQUIRED EXPORT)
-find_package(canvas_root_io REQUIRED)
+find_package(art_root_io REQUIRED)
 find_package(cetlib REQUIRED EXPORT)
 find_package(cetlib_except REQUIRED EXPORT)
 find_package(fhiclcpp REQUIRED EXPORT)
-find_package(messagefacility REQUIRED EXPORT)
+find_package(messagefacility REQUIRED)
 
 find_package(nug4 REQUIRED EXPORT)
-find_package(nugen REQUIRED EXPORT)
+find_package(nugen REQUIRED)
 find_package(nurandom REQUIRED EXPORT)
+find_package(nufinder REQUIRED EXPORT)
 find_package(nusimdata REQUIRED EXPORT)
-find_package(nutools REQUIRED EXPORT)
+find_package(nutools REQUIRED)
 
-find_package(dk2nugenie REQUIRED EXPORT)
+find_package(dk2nugenie REQUIRED EXPORT) # find module in nufinder
 find_package(dk2nudata REQUIRED EXPORT)
-find_package(ppfx REQUIRED EXPORT)
+find_package(ppfx REQUIRED)
 
-find_package(Boost REQUIRED COMPONENTS headers math_tr1 REQUIRED EXPORT) # boost::math::policies, boost::math::ellint*
+find_package(Boost REQUIRED EXPORT)
 find_package(CLHEP COMPONENTS Evaluator Geometry Random Vector REQUIRED EXPORT)
 find_package(CRY REQUIRED EXPORT) # find module in nutools
 find_package(GENIE REQUIRED EXPORT) # find module in nugen
 find_package(Geant4 REQUIRED EXPORT)
-find_package(LibXml2 REQUIRED EXPORT)
 find_package(MARLEY REQUIRED EXPORT)
-find_package(PostgreSQL REQUIRED EXPORT)
 find_package(ROOT COMPONENTS Core EG GenVector Geom Gpad Hist MathCore Matrix Physics RIO RooFitCore Tree REQUIRED EXPORT)
-find_package(Range-v3 REQUIRED EXPORT)
-find_package(SQLite3 REQUIRED EXPORT)
-find_package(ifdhc REQUIRED EXPORT)
-find_package(log4cpp REQUIRED EXPORT)
+find_package(Range-v3 REQUIRED)
+find_package(SQLite3 REQUIRED)
+find_package(ifdhc REQUIRED)
```

Comment:
I think if you restore `find_package(ifdhc REQUIRED EXPORT)`, the DUNE build will succeed.

Reply:
You're right, @lgarren.  Because `dunesim` explicitly uses the `ifdh::ifdh` target, I think my preference is to create a `dunesim` PR with an explicit `find_package(ifdhc REQUIRED)` CMake command.

I'll play around with it and let you know.

Reply:
Agreeing that dunesim needs an explicit `find_package(ifdhc REQUIRED)`.  No change needed here.

Reply:
Working on PR for dunesim.